cpe/attributes/java/java.go

```diff
@@ -90,6 +120,15 @@ func getPossibleVendors(origins []string) set.StringSet {
 	if vendorSet.Cardinality() == 0 {
 		vendorSet.Add("apache")
 	}
+
+	// Add Spring-specific vendors.
+	for name := range names {
+		if knownSpringComponents.Contains(name) {
+			vendorSet.AddAll(knownSpringVendors...)
+			break
+		}
+	}
```

Comment:
If performance is not a strong concern (it might be, though), you could also do

```suggestion
	if !names.Intersect(knownSpringComponents).IsEmpty() {
		vendorSet.AddAll(knownSpringVendors...)
	}
```
Or, to address the performance issue, maybe it would also be nice to add a `Intersects(other Set) bool` method to `pkg/set` (not in this PR, just as an idea).